### DISCUSSION TITLE
HELP! Set References inside same body, from one index to another paraghraph

### DISCUSSION THREAD

**Discussion by shahlasamadli** [xdan/jodit#1187] [Q&A] [answered]
Hi, i need set reference inside same body like that:
Lorem ipsum dolor sit amet consectetur[[**1**]].  Cupiditate voluptate labore saepe, ab maxime error recusandae minima voluptatum ipsa iste. Nihil possimus, incidunt magnam itaque vitae debitis. Vitae, ratione facilis.

References:
1. lorem ipsum skdsjds
2. another reference dhd

I use jodit editor for my react js project, please any solution, i can not find

**Comment by xdan** [comment, marked as answer]
There's no built-in footnotes/reference feature, but you can do it with plain anchor links: put `<a id="ref1"></a>` at the reference target and link to it from the text with `<a href="#ref1">1</a>` — both survive Jodit's cleanup. If you want a one-click "insert reference" button that also keeps the numbering, that's a small custom plugin/button inserting that markup. Happy to sketch it if you'd like.
